archetypes/blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: `<the title of the post>`
+date: <yyyy>-<mm>-<dd>
```

Comment:
Hi @rolfedh .
The idea is to run `hugo new` and create a Markdown file with the pre-populated front matter and possibly some shared or standard headings for a blog page. The instructions should appear as comments in the file?
The front matter could use some of the built in parameters so that the fields are filled in when the file is created, something like:

```
title: "{{ replace .Name "-" " " | title }}"
date: {{ .Date }}
draft: true
```
I don't know if the blog page always overrides the slug field.

Reply:
:+1: to templating with `draft: true`

Reply:
Thanks. Updated.

Reply:
Updated. Thanks.